### PR TITLE
updated to remove directions from existing metrics. tests updated to …

### PR DESCRIPTION
**Pull request by petertamuk**
…match

# Description

Updated metrics to match the approved spec. Directions attribute moved out to be individual metrics. 

Fixes # (issue)
https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1171

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Change required updates to testing as well. Updated and ran through test_system_metrics.py

# Does This PR Require a Core Repo Change?

- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [x] Followed the style guidelines of this project
- [x] Changelogs have been updated
- [x] Unit tests have been added
- [x] Documentation has been updated

### CODE REVIEW

**Comment by petertamuk** [comment]
Please run build at earlier convenience. Thanks

**Comment by petertamuk** [comment]
venv linting doesnt show any issues now. Should be good to go

File: instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py [hunk 1]
```diff
@@ -225,27 +240,51 @@ def _instrument(self, **kwargs):
...
 
-        if "system.disk.io" in self._config:
+        if "system.disk.io.read" in self._config:
             self._meter.create_observable_counter(
-                name="system.disk.io",
-                callbacks=[self._get_system_disk_io],
-                description="System disk IO",
+                name="system.disk.io.read",
+                callbacks=[self._get_system_disk_io_read],
+                description="System disk IO read",
```
Comment (lzchen) [inline instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py:247]: Please use the precise descriptions for each of these instruments as defined by the specification.

File: instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py [hunk 2]
```diff
@@ -473,6 +576,7 @@ def _get_system_disk_merged(
```
Comment (lzchen) [inline instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py:576]: Doesn't even look like this function is being used? Seems like the merged instrument is not supported yet. We should just remove this for the time being.

File: instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py (hunk 1 above)
Comment (petertamuk) [reply instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py:247]: @lzchen Appreciate the guidance. This is the spec I'm working off. Should I assume that metrics where the description is empty, I should leave it empty as well?

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#systemdisk---disk-controller-metrics

File: instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py [hunk 3]
```diff
@@ -91,13 +91,20 @@
...
     "system.swap.usage": ["used", "free"],
     "system.swap.utilization": ["used", "free"],
-    "system.disk.io": ["read", "write"],
-    "system.disk.operations": ["read", "write"],
-    "system.disk.time": ["read", "write"],
-    "system.network.dropped.packets": ["transmit", "receive"],
-    "system.network.packets": ["transmit", "receive"],
-    "system.network.errors": ["transmit", "receive"],
-    "system.network.io": ["transmit", "receive"],
+    "system.disk.io.read": None,
```
Comment (lzchen) [inline instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py:94]: Could you update the code example above as well in the doc string.

Comment (petertamuk) [reply instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py:103]: I believe I have addressed this with the most recent commits

File: instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py (hunk 2 above)
Comment (petertamuk) [reply instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py]: I believe I have addressed this with the most recent commits

File: instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py (hunk 1 above)
Comment (petertamuk) [reply instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py]: I believe I have addressed this with the most recent commits

**Review by lzchen — APPROVED** [review]

**Comment by petertamuk** [comment]
@lzchen I am not able to merge this. Once this passes the checks again, could you please merge this for me?

**Comment by petertamuk** [comment]
Thanks for the help!

**Comment by srikanthccv** [comment]
I believe this should be reverted https://github.com/open-telemetry/opentelemetry-specification/commit/f0c001ea285b0c6df928edc09dac031d8be8d4b9

**Comment by lzchen** [comment]
@srikanthccv 
Oh interesting. Added a tracking issue for it. https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1321

**Pull request merged**